app.rb

```diff
@@ -0,0 +1,42 @@
+require 'sinatra/base'
+require './lib/peep.rb'
+require './lib/user.rb'
+
+class Chitter < Sinatra::Base
+
+  get '/' do
+    erb(:index)
+  end
+
+  get '/peeps/new' do
+    erb(:peeps_new)
+  end
+
+  post '/peeps' do
+    Peep.add(params[:peep_content], Time.now.strftime("%I:%M %p"), User.current_user_id)
+    redirect('/peeps')
+  end
+
+  get '/peeps' do
+    @peep = Peep.all.reverse
```

Comment:
Relying on reverse may cause problems + it should not be the controller job to sort them. It would be better to have a `all_in_chronological_order` method in Peep, that returns a sorted list (maybe with the username for each, too).